### DISCUSSION TITLE
Paid tier

### DISCUSSION THREAD

**Discussion by amorriscode** [bpepple/metron#188] [API Suggestions]
Hey! 👋🏻 

I'm very excited that I discovered Metron.

I'm working on a comic book app that scratches an itch I have as a collector. To make it super useful, I'd love to integrate with an API. My biggest issue is I want the app to be commercial but I don't want to build a comic database from scratch.

Have you ever considered offering a paid tier for Metron? This tier would help support development, pay for server costs, etc. It would allow folks to increase the limits of their API keys. I would gladly pay for access to the database for my project.

I'm super excited to try Metron out and would love to contribute even if a paid tier isn't of interest. Thanks for all your work!

**Comment by bpepple** [comment]
> Have you ever considered offering a paid tier for Metron? This tier would help support development, pay for server costs, etc. It would allow folks to increase the limits of their API keys. I would gladly pay for access to the database for my project.

I'm most likely going to add an option for users to sponsor the project to defray some of the costs of running the server, but hadn't really given any thought about adding a paid tier for increased API rate limit. Right now, I'm not sure it makes much sense at the point, but itmight be worth considering in the future sometime.
